generated_code/elastic.py

```diff
@@ -62,51 +57,5 @@ def numberOfQuantities(self):
   def numberOfExtendedQuantities(self):
     return self.numberOfQuantities()
 
```

Comment:
Why is this needed/why isn't this in a super class?

Reply:
Yes we can move that to ADERDGStandard.